modules/serverless-configuring-startup-probing-serving.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative-serving/config-applications/startup-probes-for-serving.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-configuring-startup-probing-serving_{context}"]
+= Configuring startup probing
+
+For {ServerlessProductName} Serving, startup probes are not defined by default. You can define startup probes for your containers in your deployment configuration. 
+
+.Procedure
+
+* Define startup probes for your service by modifying your deployment configuration. The following example shows a configuration with two containers:
++
+.Example of defined starup probes 
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+...
+spec:
+  template:
+    spec:
+       containers:
+        - name: first-container
+          image: gcr.io/knative-samples/helloworld-go
+          ports:
+            - containerPort: 8080
+          ...
+          startupProbe: <1>
+          httpGet:
+            port: 8080
+            path: "/"
+        - name: second-container
+          image: gcr.io/knative-samples/helloworld-java
+          ...
+          startupProbe: <2>
+          httpGet:
+            port: 8080
```

Comment:
As far as I know, the sidecar containers (the second and any further container) need to use different ports. Let's use 8081 in this case.